src/Integration.Vsix/ErrorList/SinkManager.cs

```diff
@@ -67,12 +68,12 @@ public void Dispose()
             sinkRegister = null;
         }
 
-        public void AddFactory(SnapshotFactory factory)
+        public void AddFactory(ITableEntriesSnapshotFactory factory)
```

Comment:
Miscellaneous: changed the type to be the VS interface rather than our implementation of it.